src/components/ComboBox/ComboBox.js

```diff
@@ -26,6 +22,18 @@ const getInputValue = (props, state) => {
   return state.inputValue || '';
 };
 
+const findHighlightedIndex = ({ items, itemToString }, inputValue) => {
+  if (!inputValue) {
+    return -1;
+  }
+
+  return items.findIndex(item =>
```

Comment:
Seems that `Array.prototype.findIndex()` is not in [our polyfills list](https://github.com/IBM/carbon-components-react/blob/v6.41.0/.storybook/polyfills.js). We consider requirement of new polyfill as a breaking change. Do you want to in-line a (lightweight) polyfill or seek for an alternate implementation here?

Reply:
Good catch @asudoh! 🙏

Reply:
Fixed: I just updated this here https://github.com/IBM/carbon-components-react/pull/1502/commits/3ffd1406ce371ab2f17366396a0e49ad8a479ead